sanic_openapi/swagger.py

```diff
@@ -204,7 +204,7 @@ def build_spec(app, loop):
     # --------------------------------------------------------------- #
 
     _spec['definitions'] = {
-        obj.object_name: definition for cls, (obj, definition) in definitions.items()}
+        obj.object_name: definition for _, (obj, definition) in definitions.items()}
```

Comment:
After simplifying `definitions`, it can be:
```suggestion
        object_name: definition for object_name, definition in definitions.items()}
```

Reply:
`definition` is a tuple in this case, i think, so the above code won't quite work, but i can have a look at this to simplify the code a bit.
Also, it might be a good idea to sort items by object_name. What do you think?